archive/100-theorems-list/inverse-triangle-sum.lean

```diff
@@ -2,70 +2,42 @@
 Copyright (c) 2020. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jalex Stark,
+<<<<<<< HEAD
+Solving problem 42 on the list of 100 theorems.
+=======
 
 Solving problem 42 on the list of 100 theorems, which involves a "telescoping" sum.
+>>>>>>> fbfdbe1c18421daedfd6e8884c763b3098265e6c
```

Comment:
```suggestion
Authors: Jalex Stark
```
[Here's](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) the chapter of the git book that addresses merge conflicts.

Reply:
I would still remove the "Solving problem 42 ..." text from the copyright header. You might have seen some old files with that style, but we put all that info in the module docstrings now.

Reply:
Fixed, thanks. Somehow it wasn't clear to me that you were suggesting to drop both. (It looks like all of the text is highlighted the same color. I'm now guessing that color is red.)